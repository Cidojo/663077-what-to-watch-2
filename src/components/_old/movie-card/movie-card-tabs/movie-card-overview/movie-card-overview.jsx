import * as React from 'react';
import {movieCardPropTypes} from './../../../../global-custom-types';

const MovieCardOverview = (props) => {
  const {description, director, starring} = props.card;
  return (
    <div className="movie-card__text">
      <p>{description}</p>
      <p className="movie-card__director"><strong>Director: {director}</strong></p>

      <p className="movie-card__starring"><strong>Starring: {starring.join(`, `)} and
        other</strong></p>
    </div>
  );
};

MovieCardOverview.propTypes = {
  card: movieCardPropTypes
};

export {MovieCardOverview};
