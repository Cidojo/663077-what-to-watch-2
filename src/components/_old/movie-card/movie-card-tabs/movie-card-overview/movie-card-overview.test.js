import * as React from 'react';
import {MovieCardOverview} from './movie-card-overview.jsx';
import * as renderer from 'react-test-renderer';

it(`should render component without errors`, () => {

  const tree = renderer
    .create(<MovieCardOverview
      card={{
        director: ``,
        starring: [``],
        description: ``
      }}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
