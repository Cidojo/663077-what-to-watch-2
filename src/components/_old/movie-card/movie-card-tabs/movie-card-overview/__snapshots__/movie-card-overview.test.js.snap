// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component without errors 1`] = `
<div
  className="movie-card__text"
>
  <p>
    
  </p>
  <p
    className="movie-card__director"
  >
    <strong>
      Director: 
      
    </strong>
  </p>
  <p
    className="movie-card__starring"
  >
    <strong>
      Starring: 
      
       and other
    </strong>
  </p>
</div>
`;
